src/messages.h

```diff
@@ -0,0 +1,14 @@
+#ifndef MESSAGES_H
+#define MESSAGES_H
+
+#include <iostream>
+#include <string>
+#include "../include/termcolor/termcolor.hpp"
+
+// Declare color control as an external variable
+extern bool use_colors;
```

Comment:
We should avoid these kind of global variables, even more `extern` ones.

Reply:
Alright @jviotti 👍